src/components/app/App.js

```diff
@@ -12,7 +12,8 @@ import { CURRENT_AO_KEY } from '../../util/analyticalObject.js'
 import { getUrlParameter } from '../../util/requests.js'
 import { useSystemSettings } from '../SystemSettingsProvider.js'
 import AppLayout from './AppLayout.js'
-import styles from './styles/App.module.css'
+import './App.css'
```

Comment:
Possible to add these varaibles to App.module.css to keep them in one place?

Reply:
I think that global variables like the ones in App.css have to be defined in a regular css file, not a css module, which is specific to a class and doesn't have global scope.